e2e/pages/app/menu/menu.testpage.component.html

```diff
@@ -15,30 +17,95 @@
 
     <ux-menu id="menu" #menu menuClass="ux-menu-demo" [placement]="(placement$ | async)">
 
-        <button type="button" uxMenuItem>
+        <button id="menu-item-1" type="button" uxMenuItem [uxMenuTriggerFor]="MenuItemMenu">
             <span class="dropdown-menu-icon"></span>
             <span class="dropdown-menu-text">Menu Item</span>
+            <span class="dropdown-menu-right dropdown-menu-hint">
+                <ux-icon name="next"></ux-icon>
+            </span>
         </button>
 
         <ux-menu-divider></ux-menu-divider>
 
-        <button type="button" uxMenuItem>
+        <button id="menu-item-2" type="button" uxMenuItem [uxMenuTriggerFor]="MenuItemMenu">
             <span class="dropdown-menu-icon"></span>
             <span class="dropdown-menu-text">Export</span>
+            <span class="dropdown-menu-right dropdown-menu-hint">
+                <ux-icon name="next"></ux-icon>
+            </span>
         </button>
 
-        <button type="button" uxMenuItem>
+        <button id="menu-item-3" type="button" uxMenuItem  [uxMenuTriggerFor]="SaveListMenu">
             <span class="dropdown-menu-icon"></span>
             <span class="dropdown-menu-text">Save list</span>
+            <span class="dropdown-menu-right dropdown-menu-hint">
+                <ux-icon name="next"></ux-icon>
+            </span>
         </button>
 
-        <button type="button" uxMenuItem>
+        <button id="menu-item-4" type="button" uxMenuItem>
             <span class="dropdown-menu-icon"></span>
             <span class="dropdown-menu-text">Save search query</span>
         </button>
 
     </ux-menu>
 
+    <ux-menu #MenuItemMenu menuClass="mark-as-menu">
```

Comment:
```suggestion
    <ux-menu #markAsMenu menuClass="mark-as-menu">
```
The convention for property names is to use initial lowercase - check the documentation example.